sdk/provisioning/Azure.Provisioning/src/Primitives/ProvisioningConstruct.cs

```diff
@@ -45,6 +45,10 @@ public abstract class ProvisioningConstruct(ProvisioningContext? context = defau
     [EditorBrowsable(EditorBrowsableState.Never)]
     public Infrastructure? ParentInfrastructure { get; internal set; }
 
+    /// <inheritdoc />
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why is this hidden?

Reply:
It makes sense to expose publicly on `Infrastructure` but less sense on every derived type like `StorageAccount` that would only return itself.  We want this exposed for people doing really advanced things like implementing custom resolvers but to stay out of the way of people deploying something basic.